cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -179,6 +179,7 @@ func CreateStackedEtcdStaticPodManifestFile(client clientset.Interface, manifest
 // GetEtcdPodSpec returns the etcd static Pod actualized to the context of the current configuration
 // NB. GetEtcdPodSpec methods holds the information about how kubeadm creates etcd static pod manifests.
 func GetEtcdPodSpec(cfg *kubeadmapi.ClusterConfiguration, endpoint *kubeadmapi.APIEndpoint, nodeName string, initialCluster []etcdutil.Member) v1.Pod {
+	const etcdHealthEndpoint = "/health/serializable=true"
```

Comment:
I'm pretty sure it should be:
```suggestion
	const etcdHealthEndpoint = "/health?serializable=true"
```
The `serializable=true` should be in query not in path.

Reply:
doh, that makes sense.